features/fork.feature

```diff
@@ -202,3 +202,22 @@ Scenario: Related fork already exists
     And "git.my.org" is a whitelisted Enterprise host
     When I successfully run `hub fork`
     Then the url for "mislav" should be "git@git.my.org:mislav/dotfiles.git"
+
+  Scenario: --org github
+    Given the GitHub API server:
+      """
+      before {
```

Comment:
This `before` block is not needed for this scenario. It was desired for some other scenarios, but doesn't need to be copy/pasted further.